cache/models/samples.js

```diff
@@ -466,17 +467,16 @@ function upsertOneSample(sampleQueryBodyObj, isBulk, userName) {
     // add aspect name to subject set, add sample key to sample set,
     // create/update hash of sample
     return redisClient.batch([
-      ['sadd', subjKey, aspectName],
+      ['sadd', constants.indexKey.aspect, aspectName],
```

Comment:
why this change? We add aspect to aspect index when a new aspect is created. Also, where do we add aspect to subject hash when a new sample is created?